source/common/stats/isolated_store_impl.h

```diff
@@ -68,6 +71,17 @@ template <class Base> class IsolatedStatsCache {
     return *new_stat;
   }
 
+  Base& get(StatName name, bool b) {
```

Comment:
s/b/lower_case/ ? but I have some questions:

1. do we need lower-casing in the stat implementation? Or should whatever value transforms are needed be done at call-site?
2. I don't see lower-casing in the allocator_impl.cc version.

Reply:
Ah, you've found my hack. :) 

The problem is that there are multiple overloads of get() and Base constructor. I've added a dummy bool parameter to disambiguate text readouts from counters. I've initially named it |lower_case| as I thought that it could be useful, but I'm not convinced that it is, so I've named it |b| here.

Reply:
I suspected that :)

Can you just add an enum to disambiguate in a self-documenting way?

Reply:
Done